Library/Homebrew/cmd/postgresql-upgrade-database.rb

```diff
@@ -60,10 +60,10 @@ def postgresql_upgrade_database
 
     odie "No #{name} #{pg_version_data}.* version installed!" unless old_bin
 
-    server_stopped = false
-    moved_data = false
-    initdb_run = false
-    upgraded = false
+    server_stopped = T.let(false, T::Boolean)
```

Comment:
I almost wonder if we want to implement our own helper for this. I find it gross how verbose and repetitive this is. Something like `SORBET_FALSE` might be nicer.